src/tests/ftest/launch.py

```diff
@@ -1218,11 +1246,9 @@ def report_skipped_test(test_file, avocado_logs_dir, reason):
     print(message)
 
     # Generate a fake avocado results.xml file to report the skipped test.
-    # This file currently requires being placed in a job-* subdirectory.
     test_name = get_test_category(test_file)
-    time_stamp = datetime.now().strftime("%Y-%m-%dT%H.%M")
-    destination = os.path.join(
-        avocado_logs_dir, "job-{}-da03911-{}".format(time_stamp, test_name))
+    destination = os.path.join(avocado_logs_dir, "skipped_test")
+    destination = os.path.join(avocado_logs_dir, test_name)
```

Comment:
I have concerns here.  For one, I don't think you intend to overwrite the `destination` assignment, did you perhaps intend to do this (to retain the same depth of other avocado job-results)?
```suggestion
    destination = os.path.join(avocado_logs_dir, "skipped_test-{}".format(test_name))
```
My 2nd concern is that running this manually will not generate unique job-results directories per run.  Conditionalizing the code using the `args.jenkinslog` may also be needed here too.

One option that may fix both issues is to:
1) generate the fake time-stamped  job-results sub directory for this test with the constructed xml file indicating why the test was skipped (pretty much what the old code does).
2) if (1) is successful, call `rename_logs()` for this test "result" which will handle the appropriate renaming (jenkins or manual)

Something like this implemented [here ](https://github.com/daos-stack/daos/blob/2a8f96b1574a4ecc85d05393ee732642954059a7/src/tests/ftest/launch.py#L832)and [here](https://github.com/daos-stack/daos/blob/2a8f96b1574a4ecc85d05393ee732642954059a7/src/tests/ftest/launch.py#L840):
```
            if not report_skipped_test(
                    test_file["py"], avocado_logs_dir, reason):
                return_code |= 64
            else:
                rename_logs(avocado_logs_dir, test_file["py"], args)
```
This would require `rename_logs()` to know where `report_skipped_test()` wrote the results xml file.  The simple fix would be for `report_skipped_test()` to link the created directory to `latest` - which is where `rename_logs()` looks for the results.  Alternatively this path could be passed in, where it it is `None` the `latest` directory is used.

Reply:
ok, will look into it. thanks for feedback